FAQ.md

```diff
@@ -7,21 +7,26 @@
   Amplify Console Troubleshooting Guide
 </h1>
 
-### Table of contents
+## Table of contents
 
 - [Table of contents](#table-of-contents)
```

Comment:
```suggestion
```

Reply:
I know this wasn't added but thoughts on removing? Seems a bit redundant now.

Reply:
Yeah weird, I just realized it was added automatically by Visual Studio Code. I have a plugin for editing markdown files with GitHub flavor and it automatically updates the table of contents based on the page headings. Let me remove this one because I agree is redundant.